pkg/cli/workflow/unassign.go

```diff
@@ -0,0 +1,61 @@
+package workflow
+
+import (
+	"fmt"
+
+	"github.com/spf13/cobra"
+
+	"github.com/fuseml/fuseml-core/pkg/cli/client"
+	"github.com/fuseml/fuseml-core/pkg/cli/common"
+)
+
+type unassignOptions struct {
+	client.Clients
+	global         *common.GlobalOptions
+	name           string
+	codesetName    string
+	codesetProject string
+}
+
+func newUnassignOptions(o *common.GlobalOptions) *unassignOptions {
+	return &unassignOptions{global: o}
+}
+
+func newSubCmdUnassign(gOpt *common.GlobalOptions) *cobra.Command {
+	o := newUnassignOptions(gOpt)
+	cmd := &cobra.Command{
+		Use:   "unassign {-n|--name NAME} {-p|--codeset-project CODESET_PROJECT}  {-c|--codeset-name CODESET_NAME}",
+		Short: "Unassigns a workflow from a codeset",
+		Long:  `Removes the assingment between a workflow and a codeset.`,
+		Run: func(cmd *cobra.Command, args []string) {
+			common.CheckErr(o.InitializeClients(gOpt.URL, gOpt.Timeout, gOpt.Verbose))
+			common.CheckErr(o.validate())
+			common.CheckErr(o.run())
+		},
+		Args: cobra.ExactArgs(0),
+	}
+
+	cmd.Flags().StringVarP(&o.name, "name", "n", "", "name of the workflow to be unassigned")
```

Comment:
nit: it's the codeset that is unassigned from a workflow, not the other way around, isn't it ? `workflow name` is probably enough here